### ISSUE TITLE
RSS feed of activity?

### ISSUE THREAD

**Issue by BNAndras** [mradamcox/loc-insurancemaps#2]
Looking forward, volume-specific or site-wide RSS feeds would be a way to track recent activity. For larger areas of interest like Baton Rouge or New Orleans, it'd be beneficial to see if someone loaded a new volume so you can jump in. Another situation could be someone made edits to a trim mask you had previously made so you can review it. That could raise long-term engagement with the platform.

**Comment by mradamcox** [comment]
I'm not sure if RSS is the best way to do this, but overall this is a great idea. GeoNode has a good notification system (email) which could be better extended for more types of actions, and attaching notifications at the volume level is really smart. Thanks for this idea @BNAndras.

**Comment by BNAndras** [comment]
Yeah, emails would be fine here since all registered users have email addresses. The user should be able to select the frequency of the various notification types (per event, daily, or weekly). If possible, notifications should be attachable to regions as well as volumes so for example, I'd get a weekly digest for all Baton Rouge activity regardless of the specific volume involved. Otherwise, I'd need to attach notifications for each Baton Rouge volume as they're loaded, but I won't know they're loaded unless I'm checking the website regularly. That'd make the notifications fairly useless.

**Comment by mradamcox** [comment]
I'm considering the idea of having a site newsletter, which would send to registered users could opt out of, but would also be something that people could sign up for without having to register (i.e create a login/password).

This app looks worth exploring: https://github.com/jazzband/django-newsletter.

Of course there are plenty of third-party newsletter services, but integrating it directly into project would provide the very important feature of having a unified "from" email address/domain, and allow the re-use of existing transactional email configurations.

**Comment by BNAndras** [comment]
Closing this issue because the discussion moved away from the original RSS idea.

**Comment by mradamcox** [comment]
Reopening, because RSS feeds would be pretty cool.

**Comment by mradamcox** [comment]
This would be so easy to do: https://learndjango.com/tutorials/django-rss-feed-tutorial. Feeds for the following would be top priority:

`/<place-slug>/feed/rss` -  activity for any content attached to a particular locale (descendant inclusive??)
`/<map-identifier>/feed/rss` - activity related to a particular map (volume/item)

**Comment by JosiahDub** [comment]
I'll take a look right away!

**Comment by JosiahDub** [comment]
Created #344 as a draft. Ultimately I see this as similar to the Activity page. Here's my strategy:

- Get the `Place` from the slug
- Get all `Maps` associated with the `Place`
- Get all `Layers` and `Documents` with a `SessionBase`
- Sort by `date_modified` and return info about the latest X `SessionBases`

Let me know if this is on the right track.
